src/libraries/Common/src/Interop/Android/System.Security.Cryptography.Native.Android/Interop.Cipher.cs

```diff
@@ -136,6 +136,26 @@ internal static unsafe bool EvpCipherReset(SafeEvpCipherCtxHandle ctx, ReadOnlyS
             return EvpCipherFinalEx(ctx, ref MemoryMarshal.GetReference(output), out bytesWritten);
         }
 
+        [LibraryImport(Libraries.AndroidCryptoNative, EntryPoint = "AndroidCryptoNative_AeadCipherFinalEx")]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        private static unsafe partial bool EvpAeadCipherFinalEx(
+            SafeEvpCipherCtxHandle ctx,
+            byte* outm,
+            out int outl,
+            [MarshalAs(UnmanagedType.Bool)] out bool authTagMismatch);
```

Comment:
I want to call this parameter out specifically any make sure I am marshaling correctly here. The native type is `int32_t` with a value of `0` (false) and `1` (true).